src/GoogleAdWordsEventForwarder.js

```diff
@@ -53,18 +54,19 @@
 
                 try {
                     if (window.gtag && forwarderSettings.enableGtag == 'True') {
-                        if (event.CustomFlags && 
-                            Object.keys(event.CustomFlags).length &&
-                            event.CustomFlags[ENHANCED_CONVERSION_DATA]
-                        ) {
-                            if (forwarderSettings.enableEnhancedConversions === 'True') {
-                                setEnhancedConversionData(
-                                    event.CustomFlags[ENHANCED_CONVERSION_DATA]
-                                );
-                            } else {
-                                console.warn('You have a custom flag of enhanced conversions, but you have not enabled enhanced converisons');
-                            }
+                        if (hasEnhancedConversionData(event.CustomFlags)) {
+                            buildEnhancedConversionDataFunction = buildNestedEnhancedConversionData;
+                        } else if (hasStringEnhancedConversionData(event.CustomFlags)) {
+                            buildEnhancedConversionDataFunction = buildStringEnhancedConversionData;
+                        } else {
+                            console.warn('Unrecognized Custom Flags Format', event.CustomFlags);
```

Comment:
The `CustomFlags` object does not contain custom flags specific to Google Ads.  Instead, each event has all custom flags associated with it, regardless of which kit is consuming it.  This means that this else clause will warn whenever there is a non Google Ads custom flag from another kit, and this may confuse the developer. 

As an example, if a developer had both Adobe and Google Ads on here, and they log an event with both custom flags to Google Ads and Adobe, ie:
```
{
    'GoogleAds.ECData.email' = 'email@gmail.com', 
     'Adobe.PageName': 'foo',
 }

Even though there is a valid Google Ads flag, it will warn on the Adobe flag.